config/token.go

```diff
@@ -33,7 +39,7 @@ type TokenGenerated struct {
 	//
 	// example: 2018-06-20 19:40:54.116369887 +0000 UTC m=+43224.838320603
 	// required: true
-	ExpiredAt string `json:"expired_at"`
+	ExpiresOn string `json:"expiresOn"`
```

Comment:
is this our config yaml? If so, all of our yaml attribute names are snake_case, not camelCase. So if this is our yaml definition, this should be "expires_on"

Reply:
Nope, this is for the response.

Reply:
The I question why it is in the "config" package. "config" is for all our yaml config in the configmap. This should probably go somewhere else (is there anything else dealing with responses that are in "config"?

Reply:
That's because the token generation is happening on the `config` package, since before this PR. I agree that's not ideal, but changing that would make it even bigger than it is right now.

Reply:
I am going to resolve this conversation. It likely good to remove this from the config package, but it can be done outside of this PR.

Reply:
> It likely good to remove this from the config package, but it can be done outside of this PR.

is there a separate JIRA for this?